src/Chat/Plugin/Admin.php

```diff
@@ -120,4 +110,29 @@ private function parseUserProfiles(array $userProfiles): array {
 
         return $userData;
     }
+
+    /**
+     * Handle a command message
+     *
+     * @param Command $command
+     * @return \Generator
+     */
+    public function handleCommand(Command $command): \Generator
+    {
+        if (!$this->validCommand($command)) {
+            return;
+        }
+
+        yield from $this->execute($command);
+    }
+
+    /**
+     * Get a list of specific commands handled by this plugin
+     *
+     * @return string[]
+     */
+    public function getHandledCommands(): array
```

Comment:
To me that name implies it would return a list of `Command` objects, I'd actually rather go with `getHandledCommandNames()` if anything